sdcm/utils/common.py

```diff
@@ -1047,16 +1047,14 @@ def get_tables_columns_list(entity_type):
     views_list = set()
     if filter_out_mv:
         tables = get_tables_columns_list('view')
-        if not tables:
-            return []
 
         for table in tables:
             views_list.add('.'.join([name.strip() for name in table[:2]]))
         views_list = list(views_list)
 
     result = get_tables_columns_list('column')
     if not result:
-        return []
+        return result
```

Comment:
better to leave as it were, otherwise one needs to understand `get_tables_columns_list` function return values

Reply:
Fixed